server/app/views/components/FieldWithLabel.java

```diff
@@ -216,16 +210,6 @@ public FieldWithLabel setValue(OptionalLong value) {
     return this;
   }
 
-  public FieldWithLabel setValue(Currency value) {
```

Comment:
This felt unnecessary as there's only one caller and it prevented us from round-tripping the invalid state (something that isn't convertable to a currency) that may have been provided by the applicant.